7.9.вложенные_циклы.часть_2/7.9.5.py

```diff
@@ -0,0 +1,11 @@
+n = int(input())      
+
+while n > 9:          
+    s = 0    
+    while n > 0:
+        ll = n % 10  
+        s += ll   
+        n = n // 10          
```

Comment:
Не забывай о расширенных операторах присваивания:
```suggestion
        n //= 10
```